src/index.js

```diff
@@ -3,6 +3,8 @@ import $ from 'jquery';
 import Backbone from 'backbone';
 import L from 'leaflet';
 import App from './js/index';
+window.jQuery = $
```

Comment:
So this is one of those times where I knew something wasn't quite right but I was having quite a difficulty finding a better way of doing it. Without this line bootstrap will blow up. It's dependent on jQuery being accessible to it like so.